ignite/pkg/cosmostxcollector/collector.go

```diff
@@ -9,22 +9,22 @@ import (
 	"github.com/ignite/cli/ignite/pkg/cosmostxcollector/adapter"
 )
 
-// TXsCollecter defines the interface for Cosmos clients that support collection of transactions.
+// TXsCollector defines the interface for Cosmos clients that support collection of transactions.
 //
-//go:generate mockery --name TXsCollecter --filename txs_collecter.go --with-expecter
-type TXsCollecter interface {
+//go:generate mockery --name TXsCollector --filename txs_collector.go --with-expecter
+type TXsCollector interface {
```

Comment:
Just a reminder that changing this name would require a small change in the [backend](https://github.com/ignite/backend/search?q=TXsCollecter).